src/components/shared/datePicker/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import DatePicker from 'react-native-date-picker';
+
+interface DatePickerInputProps {
+  date: Date;
+  open: boolean;
+  mode?: 'date' | 'time' | 'datetime' | undefined;
+  onConfirm: () => void;
+  onCancel: () => void;
+}
+
+const DatePickerInput: React.FC<DatePickerInputProps> = ({
+  date,
+  open,
+  mode,
```

Comment:
set a default value for mode here `mode = 'date`

Reply:
Same